static/src/javascripts/projects/commercial/modules/dfp/load-advert.js

```diff
@@ -8,6 +9,9 @@ const loadAdvert = (advert: Advert): void => {
         })
         .then(() => {
             advert.startLoading();
+            return prebid.requestBids(advert);
```

Comment:
🤔 Will the return here mean that the next `.then(` (line 14) will have to wait for `prebid.requestBids(advert)` to resolve/reject?

Could `prebid.requestBids(advert)` ever error, or get stuck pending? If yes, will we make it to `window.googletag.display(advert.id)`?

Reply:
Yes, the wait is necessary, since this operation is sequential. As bad as this sounds, it is already happening, through an external js library which overrides `window.googletag.display`. This PR switches off the current system in favour of this new one (only for users in the test, and the test is opt-in only).

As for `prebid.requestBids`, it does have a timeout already. I will add a `catch` though, to be sure.